src/helpers/daily-data-providers/combined-therapy-minutes.ts

```diff
@@ -1,16 +1,17 @@
 import { appleHealthTherapyMinutesDataProvider, googleFitTherapyMinutesDataProvider } from '.';
-import MyDataHelps from '@careevolution/mydatahelps-js';
 import { DailyDataQueryResult } from '../query-daily-data';
 import { combineResults } from './common-mindful-and-therapy';
+import { getCombinedDataCollectionSettings } from './combined-data-collection-settings';
 
 export default async function (startDate: Date, endDate: Date): Promise<DailyDataQueryResult> {
     const providers: Promise<DailyDataQueryResult>[] = [];
 
-    const settings = await MyDataHelps.getDataCollectionSettings();
-    if (settings.queryableDeviceDataTypes.find(type => type.namespace == 'AppleHealth' && type.type == 'MindfulSession')) {
+    const combinedSettings = await getCombinedDataCollectionSettings();
```

Comment:
This one too...it's just adding another API call right now until this data lives in V2.  May as well just leave this as-is